### PR TITLE
Support pytorch 1.8 CI

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
       - name: Install PyTorch
         run: pip install torch==${{matrix.torch}}+cpu torchvision==${{matrix.torchvision}}+cpu -f https://download.pytorch.org/whl/torch_stable.html
       - name: Install MMCV
-        run: pip install mmcv-full==1.3.0 -f https://download.openmmlab.com/mmcv/dist/cpu/torch${{matrix.torch}}/index.html
+        run: pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cpu/torch${{matrix.torch}}/index.html
```
Comment (congee524) [inline .github/workflows/build.yml:63]: specify the version of mmcv-full may be better. In this way, there is no need to locate whether the problem lies in our code or the code of mmcv

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -84,23 +84,20 @@ jobs:
...
+            mmcv: 1.5.0+cu101
+          - torch: 1.6.0+cu101
+            torchvision: 0.7.0+cu101
+            mmcv: 1.6.0+cu101
+          - torch: 1.7.0+cu101
+            torchvision: 0.8.1+cu101
+            mmcv: 1.7.0+cu101
+          - torch: 1.8.0+cu101
+            torchvision: 0.9.0+cu101
+            mmcv: 1.8.0+cu101
```
Comment (congee524) [inline .github/workflows/build.yml:100]: Considering 3080/3090, it's necessary to check pt1.8.0 + cu111.

Comment (dreamerlin) [reply .github/workflows/build.yml:100]: We now only support cu101 arch

Comment (congee524) [reply .github/workflows/build.yml:100]: mmcv support cu111? https://download.openmmlab.com/mmcv/dist/cu111/torch1.8.0/index.html

Comment (dreamerlin) [reply .github/workflows/build.yml:100]: I mean the CUDA environment

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
Cool

**Comment by innerlee** [comment]
So we are dropping pytorch 1.3?

**Comment by innerlee** [comment]
Our installation guide still says 1.3+

**Comment by dreamerlin** [comment]
> So we are dropping pytorch 1.3?

add it back

**Pull request merged**